apis/placement/v1alpha1/override_types.go

```diff
@@ -78,10 +85,21 @@ type OverrideRule struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=20
-	// +required
-	JSONPatchOverrides []JSONPatchOverride `json:"jsonPatchOverrides"`
+	// +optional
+	JSONPatchOverrides []JSONPatchOverride `json:"jsonPatchOverrides,omitempty"`
 }
 
+// OverrideType defines the type of Override
+type OverrideType string
+
+const (
+	// JSONPatchOverrideType applies a JSON patch on the selected resources following [RFC 6902](https://datatracker.ietf.org/doc/html/rfc6902).
+	JSONPatchOverrideType OverrideType = "JSONPatch"
+
+	// DeleteOverrideType deletes the selected resources on the target clusters.
+	DeleteOverrideType OverrideType = "Delete"
```

Comment:
```suggestion
	OverrideTypeRemove OverrideType = "Remove"
```
niiiiiiiiit, how about using the same terminology as "JSONPatch"?

Reply:
What is the reason to use remove instead of delete?

The list of verbs are:
GET, POST, PUT, PATCH, DELETE

Reply:
i was looking at the JSONPatch operator (add, remove, replace)  and tried to keep them consistent.

looks like the HTTP verb is better here and we already have PATCH.